src/som/VmSettings.java

```diff
@@ -33,6 +40,16 @@
     MEMORY_TRACING = getBool("som.memoryTracing",   false);
     DISABLE_TRACE_FILE = getBool("som.disableTraceFile", false);
 
+    //disabling of message or actor tracing in general is probably not a good idea and would require to turn off other tracing as well;
```

Comment:
The comment was intended to say, that turning off actor/message tracing would be as useful as no tracing at all.